col.py

```diff
@@ -52,6 +52,10 @@ def read(self, fin):
     def __repr__(self):
         return hex(self.unknown0)+'|'+repr(self.groups)
 
+if len(sys.argv) != 2:
+    sys.stderr.write("Usage: %s <col>\n"%sys.argv[0])
+    exit(1)
+
```

Comment:
Note the `if __name__ == "__main__"` a few lines below. If that exists, the file was intended to be able to be imported in another script. By adding this "if" check and exit, you've made this impossible. Please move the checks beneath the `if __name__`.